frontend/static/style.css

```diff
@@ -185,6 +185,12 @@ img {
 	list-style-type: none;
 }
 
+.btn_padded {

```

Comment:
use inline bootstrap margin shorthand to adhere with the existing convention rather than creating a new css class